di_website/publications/models.py

```diff
@@ -418,7 +418,7 @@ class Meta:
 
     @cached_property
     def label(self):
-        return 'The summary'
+        return 'the executive summary'
```

Comment:
all lower case?

Reply:
Yeah, it gets put into "Read the executive summary"